src/main/java/com/treasuredata/android/TreasureData.java

```diff
@@ -1022,6 +1091,13 @@ public static String getSessionId(Context context) {
         return session.getId();
     }
 
+    public static void resetSessionId(Context context) {
+        Session session = getSession(context);
+        if (session != null) {
+            session.resetId();
+        }
+    }
```

Comment:
This looks fine. But just in case, `resetSessionId` is intended to be included on this merge, right?

Reply:
Yes, this is intentional